packages/fsapp/src/lib/app-router.ts

```diff
@@ -302,7 +303,10 @@ export default class AppRouter {
         }
 
         return customRoutes.filter((page: PublishedPage) => page.path !== null)
-          .reduce((ret: any, page: any) => {
+          .reduce((ret: any, page: PublishedPage) => {
```

Comment:
Still noticing an `any` here that relates back to line 263 `async fetchPageRoutes(): Promise<any>`

From what I can tell this should be typed `Recored<string, Pick<PublishedPage , 'id' | 'title'>`

Reply:
@wSedlacek changing this now, but i believe it should be:
`Record<string, Omit<PublishedPage, 'id' | 'path'>>,`

Reply:
`Omit` would exclude everything but the `id` and `path` where as on lines 310-312 it seems that we are defining `id` and `title`. That said I do see that `id`is renamed to `pageId` on those lines so the type I provided wasn't quite right on that aspect.

Sounds like you are on the right track though.